xml/System.Runtime.CompilerServices/Unsafe.xml

```diff
@@ -101,12 +122,22 @@
         <Parameter Name="elementOffset" Type="System.Int32" Index="1" FrameworkAlternate="dotnet-plat-ext-2.1;dotnet-plat-ext-2.2;net-5.0;net-6.0;netcore-1.1;netcore-3.0;netcore-3.1" />
       </Parameters>
       <Docs>
-        <typeparam name="T">The type of reference.</typeparam>
-        <param name="source">The reference to add the offset to.</param>
+        <typeparam name="T">The elemental type of the managed pointer.</typeparam>
+        <param name="source">The managed pointer to add the offset to.</param>
         <param name="elementOffset">The offset to add.</param>
-        <summary>Adds an element offset to the given reference.</summary>
-        <returns>A new reference that reflects the addition of offset to pointer.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Adds an offset to the given managed pointer.</summary>
+        <returns>A new managed pointer that reflects the addition of offset to the source pointer.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+The `elementOffset` parameter is the number of `T`-sized elements (not bytes) to add to the `source` pointer. For example, given a source pointer _ptr_ of type `ref int`, the call `Unsafe.Add<int>(ref ptr, 20)` will return a new pointer whose address points 80 bytes (= 20 elements \* 4 bytes per element) beyond _ptr_.
```

Comment:
Is it intentional we use code quotations rather than `<paramref name="" />`?

Reply:
The pattern I see elsewhere is that we use `<paramref>` for non-markdown sections and backticks for markdown sections. But @gewarren to confirm this?

Reply:
Since this is a Markdown section, the backticks are correct.